### DISCUSSION TITLE
What is the recommended "workflow" for editors/translators?

### DISCUSSION THREAD

**Discussion by hpoul** [wagtail/wagtail-localize#574] [Q&A]
Hi,
I was wondering if somebody who uses wagtail-localize in a real project has some experiences to share how to best manage translations. It's easy enough if you have content in one language completely finished and just translate that snapshot into another language. But at least for my projects, content is usually updated constantly. Is there some best practice on how to work with `wagtail-localize` in a project with editors and translators?
Do I have to tell editors that after publishing a change they have to synchronize with the translated pages, and translators have to check the "Reports" page and click through the whole list to find "Needs translation" pages? Or should I just periodically synchronize the whole Root page recursively and then look which pages which needs translation?

I think I'm missing something obvious on how to properly use `wagtail-localize` from a user point of view. the documentation only really covers the initial translation. I hope my question makes sense.

thanks,
  Herbert

**Comment by janbaykara** [comment]
Also interested in hearing peoples' workflows. Potentially this could be turned into documentation!

**Comment by Yemeni** [comment]
I was thinking about the same issue 
I want a page to be published when it's translations are ready but I can't mark it as a draft and then translate it since translation by default only gets the latest published version 

So I was thinking of creating another workflow for translations or making my own hack to fetch latest draft instead of latest published version
